lib/chef/resource/windows_package.rb

```diff
@@ -26,13 +26,12 @@ class Resource
     class WindowsPackage < Chef::Resource::Package
       include Chef::Mixin::Uris
 
-      provides :package, os: "windows"
       provides :windows_package, os: "windows"
+      provides :package, os: "windows"
```

Comment:
@randomcamel I'm not seeing where that happened--the one instance I can find looks like it wasn't removed.  Maybe a different bsd?